src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,95 @@
 import React, { Component } from 'react';
+import './NewMovie.scss';
+import PropTypes from 'prop-types';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const movie = {
+      title: this.state.title,
+      description: this.state.description,
+      imgUrl: this.state.imgUrl,
+      imdbUrl: this.state.imdbUrl,
+      imdbId: this.state.imdbId,
+    };
+
+    this.clearedForm();
+    this.props.addMovie(movie);
+  }
+
+  clearedForm = () => {
```

Comment:
```suggestion
  clearForm = () => {
```
clearForm - очистить форму
clearedForm - очищенная форма